apps/preview/next/pages/showcases/MegaMenu/BaseMegaMenu.tsx

```diff
@@ -141,17 +153,21 @@ export default function BaseMegaMenu() {
         className="flex justify-center w-full border-0 bg-primary-700 border-neutral-200 h-14 md:relative md:h-20 md:z-10"
       >
         <div className="flex items-center flex-nowrap justify-start h-full max-w-[1536px] w-full px-4 md:px-10">
-          <a href="/" aria-label="SF Homepage" className="inline-block text-white mr-2 lg:mr-10">
+          <a
+            href="/"
+            aria-label="SF Homepage"
+            className="inline-block mr-2 text-white md:mr-10 focus-visible:outline focus-visible:outline-offset focus-visible:rounded-sm"
+          >
             <picture>
-              <source srcSet={brandLogo.src} media="(min-width: 767px)" />
-              <img src={brandLogoSign.src} alt="Sf Logo" className="w-8 h-8 mr-4 md:w-[12.5rem] md:h-[1.75rem]" />
+              <source srcSet={brandLogo.src} media="(min-width: 1025px)" />
```

Comment:
```suggestion
              <source srcSet={brandLogo.src} media="(min-width: 1024px)" />
```
In order to have seamless transition on resizing we substract px on max-width so `max-width: 1023px` and `min-width: 1024px`

![image](https://user-images.githubusercontent.com/2566152/233586001-c26f782f-46c2-475f-8fd3-637cb8e2d6ae.png)

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2622/commits/2a200c8395073393be3f5853f9162e9c2d8ea3d3